src/client/public/main.css

```diff
@@ -798,4 +792,21 @@ footer {
 	width: 300px;
 }
 
+.disable-submit{
+	background: #828282 !important;
+    color: #ffffff !important;
+	border-radius: 6px !important;
```

Comment:
We repeat too much of the code. This is also a problem that I have to consider. I think that this is not necessary and we need to create basic (generic) submit button style and on specific styles we should change only the properties that really changes (for example: border radius ahouldn't change among different styles of the button, hence, we shouldn't set this property more than once). As well as other properties.

Reply:
I agree. 
From project perspctive, the additinal value of a better CSS code is really small while we do not have all pages ready... 
I think it will be fair to open an issue regarding this but acpet the situation as is right now

Reply:
https://github.com/teddy4445/lecture_website_app_generator/issues/28